Migrate docs (#3910)
kyma-project/kyma: docs/helm-broker/docs/03-04-bundles-docs.md

```

> apiVersion: cms.kyma-project.io/v1alpha1

Reviewer: user can define both the ClusterDocsTopic and DocsTopic, so you can have two kinds in one examples. Please also fix the documentation about it. Right now we are mentioning only the ClusterDocsTopic. 

Maybe we can also mention that the name of the file and place where it is stored is up to the user. It does not have to be called `docs.yaml`

    - type: {type}
      name: {name}
      mode: {mode}
      url: {{ .Values.addonsRepositoryURL }}
      filter: docs/{class_name}/ 
___
apiVersion: cms.kyma-project.io/v1alpha1
kind: DocsTopic
metadata ... 
```

> If your bundle defines `ServiceBroker`, use `DocsTopic` type. If your bundle defines `ClusterServiceBroker`, use `ClusterDocsTopic` type.

Reviewer 2: ```suggestion
If your bundle defines a ServiceBroker, use the DocsTopic type. If your bundle defines a ClusterServiceBroker, use the ClusterDocsTopic type.
```

> For detailed descriptions of all parameters, see the [ClusterDocsTopic custom resource](/components/headless-cms/#custom-resource-clusterdocstopic)

Reviewer 2: ```suggestion
For more details, see the [ClusterDocsTopic](/components/headless-cms/#custom-resource-clusterdocstopic)
```

> or [DocsTopic custom resource](/components/headless-cms/#custom-resource-docstopic).

Reviewer 2: ```suggestion
and [DocsTopic](/components/headless-cms/#custom-resource-docstopic) custom resources documentation.
```

For more information about currently supported types of the assets, read [this](/components/headless-cms/#overview-overview-headless-cms-in-kyma) document.

> One ClusterDocsTopic or DocsTopic object corresponds to a single Service Class with the same ID as the name of the specified ClusterDocsTopic or DocsTopic.

Reviewer 2: ```suggestion
One ClusterDocsTopic or DocsTopic object corresponds to a single Service Class with the same ID as the name of the specified object.
```
